techmatters/flex-plugins
# CHI1471 Additional Masking items in Flex 2.0

Primary reviewer: @stephenhand 

## Description
- secondLine of task item in 'All Tasks' section masked
- In webchat, ip address is masked. 
- Task Info Panel masking has not been straight forward. For static information with simple html code can be added/replaced. However, it has not been possible to pass the task related information to it as a react component. For now, I am leaving it as templated component.

### Checklist
- [ ] Strings are localized - can do this in a followup PR

### Related Issues
Fixes # 1471

### Verification steps
- Change permission config condition set for dev(remove "everyone" and check whether identifiers are masked for the following components/strings
 - secondLine
 - Ip address message bubble
 - Identifier in 'Info' tab
